pkg/executor/show_placement.go

```diff
@@ -268,39 +268,26 @@ func (e *ShowExec) fetchAllPlacementPolicies() error {
 }
 
 func (e *ShowExec) fetchRangesPlacementPlocy(ctx context.Context) error {
```

Comment:
Could we add a test for this change?

Reply:
PTAL @zimulala https://github.com/pingcap/tidb/pull/52254/commits/378227dd96c6eda79a769b84aef2cd329ed14116